MonoGame.Framework/Graphics/Vertices/VertexBuffer.cs

```diff
@@ -54,13 +54,18 @@ internal protected override void GraphicsDeviceResetting()
 
         public void GetData<T> (int offsetInBytes, T[] data, int startIndex, int elementCount, int vertexStride) where T : struct

```

Comment:
For now, I could add some XML doc comments to explain it in more detail? Unless you feel strongly about it, I'd prefer to stick with XNA's parameter names until there is a broader decision to move away from the XNA API.